### DISCUSSION TITLE
Disable onboard blue LED?

### DISCUSSION THREAD

**Discussion by sirbwolf** [forkineye/ESPixelStick#764] [Q&A]
Is there a way to disable the onboard blue LED on a D1 Mini?

**Comment by MartinMueller2003** [comment]
It depends. Which platform are you using?

**Comment by sirbwolf** [comment]
Using it on a ESPixelStick v3.

FW: 4.0-ci9115820340
ESP8266

**Comment by forkineye** [comment]
You cannot, as it's tied to the output data line. What you're actually seeing is it flashing the output bits so fast that it looks steady.  The only way to disable it is remove / de-solder it.
